### PR TITLE
add new csp video, the internet: how search works to video library

### PR DESCRIPTION
**Pull request by tanyaparker**
image

### CODE REVIEW

**Comment by tanyaparker** [comment]
I'm not sure why, but when I uploaded this video to aws the [download link](http://videos.code.org/2015/csp/how_search_works.mp4) doesn't work the same way as the others. Instead of immediately downloading, it takes you to a page to watch the video in full screen. But the download icon on there works... 

Can you help me figure out what changed in the amazon UI? And I can update the documentation for adding a video with their new UI.

File: pegasus/sites.v3/code.org/public/educate/resources/videos.md [hunk 1]
```diff
@@ -99,6 +99,14 @@ We are building a growing library of educational videos available for re-use by
 
 </div>
 
+<div class="col-50" style="float: left; padding: 10px;">
+
+<% facebook = {:u=>'https://youtu.be/LVV_93mBfSU'} %>
+<% twitter = {:url=>'https://youtu.be/LVV_93mBfSU', :related=>'codeorg', :text=>'THE INTERNET: How Search Works @codeorg'} %>
```
Comment (breville) [inline pegasus/sites.v3/code.org/public/educate/resources/videos.md:105]: SHOUTING? :)

Comment (tanyaparker) [reply pegasus/sites.v3/code.org/public/educate/resources/videos.md:105]: Yeah, I'm not sure who chose this but I kept it for consistency. I think it helps it to stand out more on the page. Too much?

Comment (breville) [reply pegasus/sites.v3/code.org/public/educate/resources/videos.md:105]: I guess we use all caps for a number of video captions on this page, so it's kind of the style we've chosen.  Seems like a chance to use bold if we really want to emphasize certain words, but not a big deal to leave as-is.

**Comment by breville** [comment]
It's indeed interesting to compare this new t and an existing https://videos.code.org/2015/csp/what_is_the_internet.mp4.  On both Chrome & Safari, the former streams while the latter downloads.  However, on Firefox, they both stream.

When I looked at forcing downloads long ago, I didn't find what felt an authoritative way to force a download.  That said, [this](https://stackoverflow.com/questions/21091766/mp4-download-causes-browser-to-play-file-instead-of-download) article does suggest a couple headers that can be used to force a download, and it looks like it's [possible](http://docs.aws.amazon.com/AmazonS3/latest/API/RESTObjectGET.html) to set these in S3?

All that said, looking at the headers returned for these two .mp3 files, I don't see either of those headers in play, nor any others that would obviously suggest downloading versus streaming, so I'm currently stumped.

I do wonder whether the browser uses another heuristic such as file size to decide when to stream versus download.  It's downloading the 23MB file while streaming the 71MB one.

Probably missing some crucial element, but not sure yet.  Curious if @wjordan or @ewjordan  have any ideas...

**Comment by tanyaparker** [comment]
Also, [videos.code.org/2015/csp/concept_rgb.mp4](videos.code.org/2015/csp/concept_rgb.mp4) is a 280MB video and it direct downloads.

**Pull request merged**